content/installation/from-packages.md

```diff
@@ -1,70 +1,84 @@
 ---
 layout: bt_wiki
-title: Installing using packages
+title: Installing from packages
 category: Installation
 draft: false
 weight: 200
 
 ---
 
-Compiled binaries are provided for Windows and RHEL based distributions. You can use these to easily install Cloudify.
+Cloudify packages greatly reduce the initial complexity of the installation process
+and are the recommended way of installing for new users.
+
+This installation method doesn't require internet connection during the install
+process.
+
+{{% gsNote title="Installation Environment" %}}
+Note that a virtualenv will be automatically created during installation and Cloudify's CLI will be installed within it.
+{{% /gsNote %}}
 
 ## Windows
 
-The Windows installer is a single executable which performs the following (offline) installation:
+The Windows installer is a single executable which installs the following:
 
-* Installs Python 2.7.x
-* Installs pip
-* Installs Virtualenv
-* Installs cfy
+* Python 2.7.x
+* Pip
+* Virtualenv
+* Cloudify
 
 {{% gsNote title="Installing Prerequisites" %}}
-If Python is not already installed, the executable will attempt to install it. During Python's installation you will be able to choose different installation options such as installing pip, adding the python executable to the path, and so on. If you choose, for instance, to not install pip and continue with the installation, you will be notified that you must install pip and it will be installed for you. The same goes for virtualenv.
-
 Python's installation requires a specific Microsoft Visual C++ 2008 Redistributable Package provided [here](https://www.microsoft.com/en-us/download/details.aspx?id=29). Install it if you stumble upon an error during the Python installation.
 {{% /gsNote %}}
 
-{{% gsNote title="Installation Environment" %}}
-Note that a virtualenv will be automatically created during installation and Cloudify's CLI will be installed within it.
+{{% gsNote title="Installing Python and Pip" %}}
+During Python setup wizard you will get the option to choose whether to install Pip
+with Python or not. Regardless of what you choose ,it will be installed in the next steps.
```

Comment:
should explain why it will be installed no matter what you choose. Maybe say "...steps, since it is required by Cloudify".